src/nix/modules/service/host-store.nix

```
@@ -17,6 +17,11 @@ in
       default = false;
       description = "Bind mounts the host store if enabled, avoiding copying.";
     };
+    service.hostStoreAsReadOnly = mkOption {
+      type = types.bool;
+      default = false;
```

It works on NixOS too. This should be the default.

```suggestion
      default = true;
```